RadarSDK/RadarUtils.m

```diff
@@ -164,6 +164,18 @@ + (NSString *)dictionaryToJson:(NSDictionary *)dict {
     } 
 }
 
++ (NSDictionary<NSString *, NSString *> *)extractGeofenceIdAndTimestampFromIdentifier:(NSString *)identifier {
+    NSArray<NSString *> *components = [identifier componentsSeparatedByString:@"_"];
+    if (components.count != 4) {
+        return nil; // Invalid format
+    }
+    
+    NSString *geofenceId = components[2];
+    NSString *registeredAt = components[3];
+    
+    return @{@"geofenceId": geofenceId, @"registeredAt": registeredAt};
+}
```

Comment:
I think there's a chance we graduate from using the geofence id here to using something like a "notificationId" at some point. Perhaps we'd want to future proof this method (e.g. make sure that we register geofenceId and registeredAt if the second component is "geofence"

Reply:
that's a really good point, but since this internal helper method doesn't need to be backward/forward compatible, maybe we can make that change when we graduate to a new form of identifiers to take the guesswork out of the equation?